### PR TITLE
Merge RA minelayers

### PR DESCRIPTION
**Pull request by reaperrr**
Rebased, squashed and updated with faction-specific mine appearance version of #12334.

### CODE REVIEW

File: mods/ra/rules/misc.yaml [hunk 1]
```diff
@@ -1,10 +1,17 @@
+# Only kept for backwards-compatibility with existing and imported maps, use MINV instead
 MINP:
 	Inherits: ^Mine
```
Comment (GraionDilach) [inline mods/ra/rules/misc.yaml]: Shouldn't this inherit MINV now?

**Review by GraionDilach — APPROVED** [review]
:+1: even without that nit though, but I guess the nit makes sense.

**Comment by reaperrr** [comment]
Updated.

**Review by obrakmann — APPROVED** [review]
lgtm, 👍

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/149b9b324f29ca03056aa76cea3fa1950137b9ac)

**Pull request merged**